components/brave_rewards/resources/page/components/connect_wallet_modal.tsx

```diff
@@ -33,7 +33,7 @@ function renderProviderIcon (provider: string) {
 
 type ModalState = 'info' | 'select'
 
-interface ExternalWalletProvider {
+export interface ExternalWalletProvider {
```

Comment:
Does this need to be exported? I wasn't able to find it imported anywhere.

Reply:
resolved